apps/console/src/features/roles/components/wizard/role-user-assign.tsx

```diff
@@ -579,30 +580,77 @@ export const AddRoleUsers: FunctionComponent<AddRoleUserProps> = (props: AddRole
-                                                                            />
-                                                                        </Table.Cell>
-                                                                        <Table.Cell>
-                                                                            { user.userName }
-                                                                        </Table.Cell>
-                                                                    </Table.Row>
-                                                                );
+                                                            assignedUsers?.map((user: RolesMemberInterface) => {
+                                                                if (user.orgId) {
+                                                                    return (
+                                                                        <Table.Row key={ user.value }>
+                                                                            <Table.Cell collapsing>
+                                                                                <UserAvatar
+                                                                                    data-testid={ `${ testId }-users
+                                                                                    -list-${ user.display }-avatar` }
+                                                                                    name={ 
+                                                                                        user.display
+                                                                                    }
+                                                                                    size="mini"
+                                                                                    floated="left"
+                                                                                />
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                { user.display }
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                {
+                                                                                    user.orgId
+                                                                                    === OrganizationManagementConstants
+                                                                                        .ROOT_ORGANIZATION_ID ? 
+                                                                                        "Asgardeo" : user.orgName
```

Comment:
Let's remove this hardcoded values.

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/3887/commits/50e5e9f56cda09b88200af2487ac61e9ebf43808

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/3887/commits/969313ac6c03dbca237c7f22c785b9ad2d35bd99